integration/integration_test.go

```diff
@@ -61,7 +61,7 @@ func TestDefaultIntegrationWritesToStdout(t *testing.T) {
 	f.Close()
 	payload, err := ioutil.ReadFile(f.Name())
 	assert.NoError(t, err)
-	assert.Equal(t, `{"name":"integration","protocol_version":"2","integration_version":"4.0","data":[]}`, string(payload))
+	assert.Equal(t, "{\"name\":\"integration\",\"protocol_version\":\"2\",\"integration_version\":\"4.0\",\"data\":[]}\n", string(payload))
```

Comment:
as it's just a test, to concat with `+ "\n"` would be more readable

Reply:
wdyt? https://github.com/newrelic/infra-integrations-sdk/pull/152

Reply:
I agree